src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/Visitors/CSharpTagHelperFieldDeclarationVisitor.cs

```diff
@@ -46,13 +46,18 @@ protected override void Visit(TagHelperChunk chunk)
                                       CSharpTagHelperCodeRenderer.ExecutionContextVariableName,
                                       value: null);
 
-                    WritePrivateField(_tagHelperContext.RunnerTypeName,
-                                      CSharpTagHelperCodeRenderer.RunnerVariableName,
-                                      "new " + _tagHelperContext.RunnerTypeName + "()");
+                    Writer.Write("private ")
+                          .WriteVariableDeclaration(
+                        _tagHelperContext.RunnerTypeName,
+                        CSharpTagHelperCodeRenderer.RunnerVariableName,
+                        value: null);
```

Comment:
nit: very strange indentation.  hit <Enter> after `Writer` and let VS line the rest up.